### PR TITLE
feat: add Microsoft plugin setup

### PR DESCRIPTION
**Pull request by alorle**

### CODE REVIEW

File: private-plugins
Comment (MIGUELez11) [inline private-plugins:1]: Don't commit the private-plugins, it's very error prone due to hard to solve conflicts.

We usually only commit the private-plugins after squashing the private-plugins branch and we are merging to dev, made by the reviewer

Comment (alorle) [reply private-plugins:1]: Ok!! Will fix it later!!

Comment (MIGUELez11) [reply private-plugins:1]: No problem 😜

Comment (alorle) [reply private-plugins:1]: Done!!

**Review by MIGUELez11 — APPROVED** [review]

**Comment by MIGUELez11** [comment]
Okay, then you're free to merge to your own branch

**Pull request merged**